docs/otlp/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Clear default log providers added by the host.
+builder.Logging.ClearProviders();
```

Comment:
It actually makes the `dotnet run` experience really bad since you won't get a console output that tells you the endpoint of your ASP.NET app. This is what you get:

![image](https://github.com/user-attachments/assets/2a445b6d-1ba3-48a5-966f-3126bea945f1)

Reply:
Removed